website/snippets/_sl-partner-links.md

```diff
@@ -1,11 +1,23 @@
 <!-- turn this list into cards or sections once more docs are provided -->
-The dbt Semantic Layer integrations are capable of querying dbt metrics, importing definitions, surfacing the underlying data in partner tools, and more.  These are the following tools that integrate with the dbt Semantic Layer:
+## Integrations
 
-1. **Mode** &mdash; To learn more about integrating with Mode, check out their [documentation](https://mode.com/help/articles/supported-databases/#dbt-semantic-layer) for more info.
-2. **Hex** &mdash; To learn more about integrating with Hex, check out their [documentation](https://learn.hex.tech/docs/connect-to-data/data-connections/dbt-integration#dbt-semantic-layer-integration) for more info. Additionally, refer to [dbt Semantic Layer cells](https://learn.hex.tech/docs/logic-cell-types/transform-cells/dbt-metrics-cells) to set up SQL cells in Hex.
-3. **Google Sheets** &mdash; Google Sheets integration coming soon.
-4. **Tools that allows you to write SQL** &mdash; They must meet one of the two criteria: 
-    * Supports a generic JDBC driver option (such as DataGrip) or 
-    * Supports Dremio and uses ArrowFlightSQL driver version 12.0.0 or higher.
+The dbt Semantic Layer integrations can do things like query dbt metrics, import definitions, surface the underlying data in partner tools, and more.
+
+The tools that work with the dbt Semantic Layer include:
+
+1. **Mode**<br />
+To learn more about integrating with Mode, check out their [documentation](https://mode.com/help/articles/supported-databases/#dbt-semantic-layer) for more info.
+
+1. **Hex**<br />
+To learn more about integrating with Hex, check out their [documentation](https://learn.hex.tech/docs/connect-to-data/data-connections/dbt-integration#dbt-semantic-layer-integration) for more info. Additionally, refer to [dbt Semantic Layer cells](https://learn.hex.tech/docs/logic-cell-types/transform-cells/dbt-metrics-cells) to set up SQL cells in Hex.
+
+1. **Google Sheets**<br />
+Google Sheets integration coming soon.
+
+1. **Tools that allows you to write SQL**<br />
+To connect to tools that allow you to write SQL, they must meet one of the two criteria: 
+   - Supports a generic JDBC driver option (such as DataGrip) or 
+   - Supports Dremio and uses ArrowFlightSQL driver version 12.0.0 or higher.
```

Comment:
```suggestion
   - Uses Arrow Flight SQL driver version 12.0.0 or higher.
```
I would remove the Dremio part. While they also use AFS, we really have two separate products.